disagmoe/frontend/engine.py

```diff
@@ -349,8 +349,6 @@ def _cuda_graph_capture_attn(self):
                 graph.replay()
     
     def _cuda_graph_capture_experts(self):
-        for _ in range(5):
-            self.static_output = self.executor.execute(0, self.static_input, self.static_batch_sizes)
```

Comment:
The expert step bug mentioned in https://github.com/USC-NSL/DisagMoE/issues/40 is caused by using layer 0 for every expert worker. Using self.model_config.layers[0] may help to fix it.

Reply:
Since we have already had the expert warmup before capturing cuda graph, the warmup here is no longer required. Therefore, removed.

Reply:
Yea, perhaps we can add it in https://github.com/USC-NSL/DisagMoE/pull/41

Reply:
Updated in #41